similarity-tester/main.py

```diff
@@ -0,0 +1,96 @@
+from typing import List
+import yaml
+import openapi_client
+from openapi_client.models.search_similar_request import SearchSimilarRequest
+
+configuration = openapi_client.Configuration(
+    host = "http://localhost:8100"
+)
+
+def get_similar(question: str) -> List[dict]:
+    """
+    Sends a request to the LM module API to find the nearest knowledge base elements for a given question
+
+    :param question: The question for which the nearest ones will be searched in the knowledge base
+    :return:
+    """
+    with openapi_client.ApiClient(configuration) as api_client:
+        api_instance = openapi_client.QuestionsApi(api_client)
+
+        try:
+            api_response = api_instance.api_process_questions_process_questions_post(
+                SearchSimilarRequest.from_dict({"question": question})
+            )
+            return api_response.to_dict()['result']
+        except Exception as e:
+            print("Exception when calling QuestionsApi->api_process_questions_process_questions_post: %s\n" % e)
+
+
+def should_include_wrapper(inclusion_list: List[str]):
+    def should_include(contexts: List[dict]):
+        contexts_questions = [i["question"].strip() for i in contexts]
+        contexts_set = {c for c in contexts_questions}
+
+        nonlocal inclusion_list
+
+        for question in inclusion_list:
+            if question.strip() not in contexts_set:
+                raise Exception(f"There is no question \"{question}\" in context {contexts_questions}")
```

Comment:
Ну если используешь исключения, давай кидать конкретный класс, а не общий. Ну и можно заменить на assert:

```suggestion
            assert question.strip() in contexts_set, f"There is no question \"{question}\" in context {contexts_questions}"
```